Chapter Zero PR (#191)
johko/computer-vision-course: chapters/en/Unit 0 - Welcome/learn.mdx

# What you will  learn

> The course consists of theory, hands-on tutorials, and challenges.

Reviewer: ```suggestion
The course is composed of theory, practical tutorials, and engaging challenges.
```

Reviewer: Couple of nits

> * **Theory Part** : here you will learn the theoretical concepts of computer vision in detail using practical examples.

Reviewer: ```suggestion
* **Theory Part** : This section covers the theoretical principles of computer vision, explained in detail with practical examples.
```

Reviewer: Updated for better readability.

> * **Hands-on Tutorials** : learn to train and implement foundational and well-known computer vision models using Google Colab notebooks.

Reviewer: ```suggestion
* **Hands-on Tutorials** : You will learn how to train and apply key computer vision models using Google Colab notebooks.
```

Reviewer: made a bit more "high leveel" for those learners who are not yet familiar with models...

> * **Challenges** : put your knowledge to the test with quizzes and assignments, offering an opportunity to apply what you've learned.

Reviewer: ```suggestion
* **Challenges** : Test your understanding with quizzes and assignments, allowing you to put into practice what you have learned.
```

Reviewer: Reworded for better readability

Reviewer 2: Merged all your suggestions and Thanks for the enhancements @ATaylorAerospace 🤗.

> * **Challenges** : Test your understanding with quizzes and assignments, allowing you to put into practice what you have learned.

Reviewer 3: I think as of now we don't have this so it's best to remove

> To illustrate what these computer vision models can achieve, here's a simple demo of a cat vs. dog classifier created with Gradio.

Reviewer 3: maybe you can give a link to Gradio

Author: A link to the gradio with demo? or a link to Gradio the package?

<iframe
	src="https://huggingface.co/spaces/ak0601/cat_dog_classifier"
	frameborder="0"
	width="850"
	height="450">
</iframe>

## Course Structure

The course is organized into multiple units, covering the fundamentals and delving into in-depth exploration of state-of-the-art models.

> * **Unit 1 - Fundamentals of Computer Vision** : this unit covers the essential concepts to get started with computer vision: the need for computer vision, the field's basics, and its applications. Explore image fundamentals, formation, and preprocessing, along with learning key aspects of feature extraction.

Reviewer 4: ```suggestion
* **Unit 1 - Fundamentals of Computer Vision** : this unit covers the essential concepts to get started with computer vision: the need for computer vision, the field's basics, and its applications. Explore image fundamentals, formation, and preprocessing, along with key aspects of feature extraction.
```
